deploy/README.md

```diff
@@ -0,0 +1,33 @@
+# ECK Helm Chart
+
+This directory contains the experimental Helm chart for deploying ECK.
+
+## Usage
+
+Install the CRDs and deploy the operator with cluster-wide permissions to manage all namespaces.
+
+```sh
+helm install elastic-operator eck -n elastic-system --create-namespace 
```

Comment:
It is called `eck`. The `elastic-operator` value in that command is the name of the release. You can install a chart multiple times with different release names.

Reply:
I would be cautiously in favour of calling it something suffixed by `-operator` either `eck-operator` or `elastic-operator` and if need be adjust the naming of the sub-chart. Just because `eck` would be more than just the operator.

Reply:
I have renamed the chart to `eck-operator`.